README.md

```diff
@@ -15,14 +15,14 @@
 
 ```sh
 # Development
-DEV_ETHEREUM_PROVIDER=http://localhost:8545
-DEV_STORE_PROVIDER=https://kleros.in
-DEV_ARBITRATOR_ADDRESS=0xaea35f89f98996ae06aac344ab4b9ce1731059c4
+REACT_APP_DEV_ETHEREUM_PROVIDER=http://localhost:8545
+REACT_APP_DEV_STORE_PROVIDER=https://kleros.in
+REACT_APP_DEV_ARBITRATOR_ADDRESS=0xaea35f89f98996ae06aac344ab4b9ce1731059c4
```

Comment:
This address has changed. I think you might just want to leave this blank in the boilerplate as it is likely to change a lot